docs/sources/flow/reference/components/otelcol.exporter.debug.md

```diff
@@ -0,0 +1,88 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/otelcol.exporter.debug/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/otelcol.exporter.debug/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/otelcol.exporter.debug/
+- /docs/grafana-cloud/send-data/agent/flow/reference/components/otelcol.exporter.debug/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.exporter.debug/
+description: Learn about otelcol.exporter.debug
+labels:
+  stage: experimental
+title: otelcol.exporter.debug
+---
+
+# otelcol.exporter.debug
+
+`otelcol.exporter.debug` accepts telemetry data from other `otelcol` components and writes them to the console (stderr).
+You can control the verbosity of the logs.
+
+{{< admonition type="note" >}}
+`otelcol.exporter.debug` is a wrapper over the upstream OpenTelemetry Collector `debug` exporter. 
+If necessary, bug reports or feature requests are redirected to the upstream repository.
+{{< /admonition >}}
+
+Multiple `otelcol.exporter.debug` components can be specified by giving them different labels.
+
+## Usage
+
+```river
+otelcol.exporter.debug "LABEL" { }
+```
+
+## Arguments
+
+`otelcol.exporter.debug` supports the following arguments:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`verbosity`           | `string` | Verbosity of the generated logs. | `"basic"` | no
```

Comment:
```suggestion
`verbosity`           | `string` | Verbosity of the generated logs. | `"normal"` | no
```
The default [upstream](https://github.com/open-telemetry/opentelemetry-collector/tree/v0.86.0/exporter/debugexporter) is "normal", so let's use the same one here.